cmd/root.go

```diff
@@ -86,11 +86,16 @@ var RootCmd = &cobra.Command{
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
 	if err := RootCmd.Execute(); err != nil {
-		logrus.Error(err.Error())
+		code := -1
+		var logger logrus.FieldLogger = logrus.StandardLogger()
 		if e, ok := err.(ExitCode); ok {
-			os.Exit(e.Code)
+			code = e.Code
+			if e.Message != "" {
+				logger = logger.WithField("error", e.Message)
+			}
 		}
-		os.Exit(-1)
+		logger.Error(err)
+		os.Exit(code)
```

Comment:
what was previously called internally and externally `hint` is now called `Message` here but is shown as `error` to the user. I would highly prefer it continues to be called `hint` everywhere as it's basically that .. us trying to hint to the user what went wrong

Reply:
Fixed

Reply:
Can you also name name the struct field `Hint` instead of `Message`

Reply:
Fixed